x-pack/plugins/security_solution/common/endpoint/types/actions.ts

```diff
@@ -53,9 +53,15 @@ export interface KillProcessActionOutputContent {
 
 export interface ResponseActionGetFileOutputContent {
   code: string;
-  path: string;
-  size: number;
   zip_size: number;
+  /** The contents of the zip file. One entry per file */
```

Comment:
Its a `zip` file, so it could have multiple files in the future. Today, its just one. Also, this `contents` array is the same one that is sent to the file storage API and stored with the File metadata - we just have a copy of it now in our Action Response as well.